src/style/components/_LocationSlider.scss

```diff
@@ -0,0 +1,55 @@
+@use "../variables" as v;
+
+.slider-container {
+  margin: 1rem;
+  padding: 1rem;
+}
+
+input[type="range"] {
```

Comment:
you should check some of the scss stuff you can do here.
https://sass-lang.com/documentation/style-rules/declarations#interpolation